test/validation/api/classification/odp_classification_common.c

```diff
@@ -496,6 +508,15 @@ odp_packet_t create_packet(cls_packet_info_t pkt_info)
 			ODPH_ERR("odph_udp_tcp_chksum failed\n");
 			return ODP_PACKET_INVALID;
 		}
+	} else if (pkt_info.l4_type == CLS_PKT_L4_AH) {
+		ah->next_header = ODPH_IPV4;
+		ah->ah_len = ODPH_AHHDR_LEN;
+		ah->pad = 0;
+		ah->spi = 0;
```

Comment:
Maybe the default value could be e.g. 256 (zero is reserved for local use and should not appear in the wire and the rang 1-255 is reserved for future use). Then maybe the actual SPI used in the negative test could also be something else than 1 (e.g. val + 1). And the same comment for ESP.